utility/utils.py

```diff
@@ -728,6 +729,7 @@ def destroy_cluster(cluster_path):
 def get_openshift_installer(
     version=DEPLOYMENT['installer_version'],
     bin_dir=RUN['bin_dir'],
+    force_download=False,
```

Comment:
I think you can get rid of default 'False' here, since its always set by default.

Reply:
@vasukulkarni, I'm not sure if I understand what you meant... what is the problem to have default `False` here?

Reply:
@dahorak what I meant was the default value of True is set in config.yaml and seeing False in function initialization confused me a bit, I think we can safely remove this initialization.

Reply:
The reason, why it is set to `False` here and to `True` in the default configuration is, that this function (and mainly the second for getting client) might be used multiple times during any test run, to get the path of the required binary, so it doesn't make sense to download everything again multiple times, if it already exists. The variable in `default_config.yaml` is meant just for deployment - during which on the other hand make sense to force the download, to be sure it is used the proper version. With this in mind, it probably make sense, to move the `force_download_client` variable also to the `DEPLOYMENT` section...
Also I can't remove the initialization, since there are already two arguments with initialization before and I don't want to reorder the parameters of those functions.

Reply:
> With this in mind, it probably make sense, to move the `force_download_client` variable also to the `DEPLOYMENT` section...

Done in fc02eb0.